contract/src/lib.rs

```diff
@@ -124,6 +131,12 @@ impl Crossword {
     }
 }
 
+impl Default for Crossword {
+    fn default() -> Self {
+        Self { puzzles: LookupMap::new(b"c") }
```

Comment:
I'm not sure what we should use for a `key_prefix` so I just put `b"c"` arbitrarily